files/jcarousel/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "jcarousel",
+  "repo": "jsor/jcarousel",
+  "files": {
+    "include": ["dist/jquery.jcarousel.min.js"]
```

Comment:
Add a basePath and there are a number more files that probably should be included based on past release
https://github.com/anshumanf/jsdelivr/tree/patch-3/files/jcarousel/0.3rc1